var/spack/repos/builtin.mock/packages/cmake-client/package.py

```diff
@@ -91,9 +91,13 @@ def install(self, spec, prefix):
         # check that cmake is in the global scope.
         global cmake
         check(cmake is not None, "No cmake was in environment!")
-
+        print(spec['cmake'].prefix)
+        print(type(spec['cmake']))
+        print('-----------------')
```

Comment:
(minor) I'm assuming these print statements are for debugging and can be removed.